interpreter/llvm/src/cmake/config-ix.cmake

```diff
@@ -372,6 +372,8 @@ elseif (LLVM_NATIVE_ARCH MATCHES "hexagon")
   set(LLVM_NATIVE_ARCH Hexagon)
 elseif (LLVM_NATIVE_ARCH MATCHES "s390x")
   set(LLVM_NATIVE_ARCH SystemZ)
+elseif (LLVM_NATIVE_ARCH MATCHES "s390")
```

Comment:
Hi Mattias @ellert - the llvm people tell us at https://reviews.llvm.org/D33465 that this is wrong. We'll back this out if you don't object within the next couple of days... I hope you understand - if llvm explicitly says "wrong" it doesn't make much sense for us to overrule them.